content/doc/administration/requirements/windows.adoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: documentation
+title:  Windows Support Policy
+---
+
+This page documents the Windows support policy for the Jenkins server and agents.
+
+== Scope
+
+Jenkins plugins, e.g. plugin:windows-slaves[WMI Windows Agents],
+may set additional requirements to Windows versions on masters and/or agents.
+This page does not document such requirements,
+please refer to plugin documentation.
+
+== Why?
+
+Theoretically Jenkins can run everywhere where you can run a supported Java version,
+but there are some limitations in practice.
+Jenkins core and some plugins include native code or depend on Windows API and subsystems,
+and hence they rely on specific Windows platforms and versions.
+In the case of Windows services we also use link:https://github.com/winsw/winsw[Windows Service Wrapper (WinSW)] which requires .NET Framework.
+
+== Support levels
+
+We define multiple support levels for Windows platforms.
+
+[width="100%",cols="20%,40%,40%",options="header",]
+|===
+|Support level |Description |Platforms
+
+| **Level 1** - Full support
+| We run automated testing for these platforms, and we intend to timely fix the reported issues.
+a| * 64-bit versions of latest Windows and Windows Server versions,
+    with the latest GA update pack
+  * Windows versions used in the official Docker images
+
+| **Level 2** - Supported
+| We do not actively test it, but we intend to keep compatibility.
+  We are happy to accept patches.
+a|
+  * 64-bit Windows 10 versions generally supported by Microsoft
+  * 64-bit Windows Server versions generally supported by Microsoft
+
+| **Level 3** - Best effort
```

Comment:
"Best effort" seems the wrong wording here.
not sure what is better but best effort is level 1, this is least effort before way say unsupported

Reply:
I am used to "best effort, no guarantees" phrasing for such kind of support. No strong opinion, would appreciate feedback from @jenkins-infra/copy-editors and native speakers

Reply:
```suggestion
| **Level 3** - Patches considered
```